internal/pgadmin/crypto.go

```diff
@@ -0,0 +1,150 @@
+package pgadmin
```

Comment:
I would love it if there were a test of the functions in this file. Perhaps a simple `decrypt(encrypt(orig, key), key) == orig` covers it?

Not a blocker.

Reply:
I would a lot more tests in general :grin:

Reply:
You'll both be happy to know there are unit tests for it, in the PCF package from which this source file originated. I'll pull it over once I review it to see why I didn't the first time.

Reply:
OK for "post-release"

My test comments was about the overall project.